src/marshmallow/schema.py

```diff
@@ -878,7 +875,9 @@ def __apply_nested_option(self, option_name, field_names, set_operation):
             setattr(self.declared_fields[key], option_name, new_options)
 
     def _init_fields(self):
-        """Update fields based on schema options."""
+        """Update self.fields, self.load_fields, and self.dump_fields based on schema options.
+        This method is private API.
```

Comment:
This was always private even though we don't actually use `__`, so I don't consider this a breaking change.